.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,40 @@
+name: Spellcheck
+on: 
+  push:
+    # branches: [main]
```

Comment:
We might want to add path filtering, so we only trigger this change on `*.md` file changes. That way we don't run this unnecessarily if we just change some TS or Go files. You can take a look at other workflows as to how its done.

Reply:
sure ...will look into it and try to add some filters